src/test/matrixHelpers.ts

```diff
@@ -0,0 +1,103 @@
+import { validatedFetch } from "@eng-automation/js";
+import Joi from "joi";
+
+
+const accessTokenResponseSchema = Joi.object({ acces_token: Joi.string() });
```

Comment:
```suggestion
const accessTokenResponseSchema = Joi.object<accessTokenResponse>({ access_token: Joi.string() });
```

Reply:
Does it bring anything aside from more characters?

Reply:
You mean the `<accessTokenResponse>` part? It prevents the `acces_token` typo from happening, right now it fails silently.

Reply:
Armed with a "wait, how on earth did it work???" question, I realised that I forgot to put `.required()` to schemas...

Reply:
![nice catch](https://github.com/paritytech/polkadot-testnet-faucet/assets/588262/1ceebe2d-f77d-4d49-8f7b-43aacd155542)